packages/components/src/docs/General/10-FAQ.mdx

```diff
@@ -0,0 +1,71 @@
+3. Samples: These are practical demonstrations showcasing the usage of our components and styles within specific contexts.
+4. Patterns: These are predefined use-cases designed to address specific design requirements without the need for a dedicated component.
+Visit our [introduction page](https://solid-design-system.fe.union-investment.de/x.x.x/storybook/index.html?path=/docs/docs-general-introduction--docs#structure) for more details.
+
+### Which browsers are supported?
+TBA
+
+### How can I submit a feature request or report a bug?
+To do so, navigate to the issue section on GitHub and utilize our provided issue templates. Whether you're suggesting a new feature or reporting a bug, these templates offer placeholders and designated content sections where you can input the necessary information. Please ensure to include as many detailed specifics as possible when submitting your request or bug report.
+
+### Is there an endpoint to the development of the Solid Design System?
+No, the Solid Design System is an ongoing endeavor, much like the ever-evolving nature of the web itself. We are committed to continually refining and expanding its features, ensuring it remains aligned with the latest standards and technologies.
+
+### Is there a roadmap for upcoming features?
+Sure. You can explore our project board for a detailed overview of our [roadmap](https://github.com/orgs/solid-design-system/projects/1/views/48).
+
+### What we do and what we don't do?
+- ✅ Provide guidance and support with components, styles, samples and patterns
+- ✅ Create, update and maintain our components, styles, samples and patterns
+- ✅ Write documentation about the system’s elements’ behaviours and interactions
+- tba
+
+- ❌ Screen designs
+- ❌ Prototype flows
+- ❌ Specify your product's rules
+- ❌ Maintain business area libraries
+- tba
+
+### Does the use of the Solid Design System guarantee my website's accessibility?
+No, while our components, styles, samples, and patterns are crafted to fulfill WCAG 2.2 specifications independently, they do not ensure complete web accessibility on their own. Our offerings constitute only a portion of your website or application. Your usage, content, and overall structure play significant roles in achieving full accessibility. We aim to provide flexibility while encouraging you to consider these aspects to deliver a fully accessible experience.
+
+### Who is working on the Solid Design System?
+The Solid Design System team, consisting of Product Owner, Brand Designer, UX Designers, Developers and Architects ....
+
+### How do I reach out to the Solid Design System team?
+For any inquiries specific to the library, feel free to open an [issue](https://github.com/solid-design-system/solid/issues) or initiate a [discussion](https://github.com/solid-design-system/solid/discussions) on GitHub. If you need to directly contact the core team, you can also reach us at sds@union-investment.de.
+
+## Technical questions:
+
```

Comment:
At least in this section I would like to read sth about design tokens and their value.

See as ref.: https://www.figma.com/file/VTztxQ5pWG7ARg8hCX6PfR/Solid-DS-%E2%80%93-Component-Library?type=design&node-id=7355-30310&mode=design&t=TExgQe8xUG748nAL-11

Reply:
IMO this is part of the contridbuting guide and the documentation in Storybook and/or Figma. Don't really see this here as it is more technincal documentation rather than a question in this section...

@mariohamann what do you think?

Reply:
Hm. I get @coraliefeil says – I still experience that people don't understand the concept of Design tokens, and they can be used by any team at Union Investment – designers and devs! Therefore I think it could be good to give some reasoning here.

Reply:
I added another section about the tokens.